### ISSUE TITLE
Incompatible version on iOS with rnmapbox-maps

### ISSUE THREAD

**Issue by michelebanfi** [rnmapbox/maps#3060]
## Environment
- Dev OS: OSX 12.3.1

## Steps to reproduce

Update rnmapbox-maps to 10.15.0 and run `pod install`

```
[!] CocoaPods could not find compatible versions for pod "MapboxMaps":
  In snapshot (Podfile.lock):
    MapboxMaps (= 10.15.0, ~> 10.15.0)

  In Podfile:
    rnmapbox-maps (from `../node_modules/@rnmapbox/maps`) was resolved to 10.0.15, which depends on
      MapboxMaps (~> 10.16.0)
```

**Comment by mfazekas** [comment]
[use `pod update MapboxMaps`](https://github.com/rnmapbox/maps/blob/main/ios/install.md#pod-install-fails-on-upgrade-of-rnmapboxmaps-with-could-not-find-compatible-versions-for-pod-mapboxmaps)

**Comment by rokaiser** [comment]
Hello @mfazekas, It seems I have the same issue, but since I am using expo EAS I don't have the pod commands. I get this error:

```
[!] CocoaPods could not find compatible versions for pod "MapboxMaps":

  In Podfile:

    rnmapbox-maps (from `../node_modules/@rnmapbox/maps`) was resolved to 10.0.15, which depends on

      MapboxMaps (= 10.XX.XX)
```

Other points that might be relevant:

- package.json dependency:

`"@rnmapbox/maps": "^10.0.15",`

- package-locl.json
   ```
 "@rnmapbox/maps": {
      "version": "10.0.15",
      "resolved": "https://registry.npmjs.org/@rnmapbox/maps/-/maps-10.0.15.tgz",
      "integrity": "Ithinkitsanotherkey",
      "requires": {
        "@turf/along": "6.5.0",
        "@turf/distance": "6.5.0",
        "@turf/helpers": "6.5.0",
        "@turf/length": "6.5.0",
        "@turf/nearest-point-on-line": "6.5.0",
        "@types/geojson": "^7946.0.7",
        "debounce": "^1.2.0"
      }

```
- App.json file plugin
```
"plugins": [
      [
        "@rnmapbox/maps",
        {
          "RNMapboxMapsImpl": "mapbox",
          "RNMapboxMapsVersion": "10.XX.XX",
          "RNMapboxMapsDownloadToken": "mykey"
        }
      ]
```

I have tried almost everything but since I am new to react native, I am probably missing a very elementary thing.
It would be awesome if you could help me :) 
Thanks a lot!
